src/commands/start-client.js

```diff
@@ -137,7 +137,15 @@ module.exports = function (buildOnly) {
   }
   else {
     logger.info("Bundling assets…");
-    compiler.run(() => {
+    compiler.run((error, stats) => {
+      const errors = stats.toJson().errors;
+      if (errors.length) {
+        errors.forEach((e) => {
+          logger.error(e);
+        });
+        return;
+      }
+
```

Comment:
I'm not sure under what conditions we'd get warnings, but maybe we'd want to logger.warn `stats.toJson().warnings` as well?